examples/if_this_then_that/ifttt_sports_flask.py

```diff
@@ -0,0 +1,198 @@
+score update for the team you specify. Instructions below will lead you through setting
+up an applet on the IFTTT website. When the applet trigger is called (which sends a web
+request received by the flask server started in this example), Cozmo will play an
+animation, show an image on his face, and speak the in-game update.
+
+Please place Cozmo on the charger for this example. When necessary, he will be
+rolled off and back on.
+
+Follow these steps to set up and run the example:
+    1) Provide a a static ip, URL or similar that can be reached from the "If This
+        Then That" server. One easy way to do this is with ngrok, which sets up
+        a secure tunnel to localhost running on your machine.
+
+        To set up ngrok:
+        a) Follow instructions here to download and install:
+            https://ngrok.com/download
+        b) Run this command to create a secure public URL for port 8080:
+            ./ngrok http 8080
+        c) Note the HTTP forwarding address shown in the terminal (e.g., http://55e57164.ngrok.io).
+            You will use this address in your applet, below.
+
+        WARNING: Using ngrok exposes your local web server to the internet. See the ngrok
+        documentation for more information: https://ngrok.com/docs
+
+    2) Set up your applet on the "If This Then That" website.
+        a) Sign up and sign into https://ifttt.com
+        b) Create an applet: https://ifttt.com/create
+        c) Set up your trigger.
+            1. Click "this".
+            2. Select "ESPN" as your service.
+            3. Under "Choose a Trigger", select “New in-game update".
+            4. In section "Complete Trigger Fields", enter your sport and team and click “Create Trigger".
+
+        d) Set up your action.
+            1. Click “that".
+            2. Select “Maker" to set it as your action channel. Connect to the Maker channel if prompted.
+            3. Click “Make a web request" and fill out the fields as follows. Remember your publicly
+                accessible URL from above (e.g., http://55e57164.ngrok.io) and use it in the URL field,
+                followed by "/iftttSports" as shown below:
+
+                 URL: http://55e57164.ngrok.io/iftttSports
+                 Method: POST
+                 Content Type: application/json
+                 Body: {"AlertBody":"{{AlertBody}}"}
+
+            5. Click “Create Action" then “Finish".
+
+    3) Test your applet.
+        a) Run this script at the command line: ./ifttt_sports.py
+        b) On ifttt.com, on your applet page, click “Check now”. See that IFTTT confirms that the applet
+            was checked.
+        c) Wait for new in-game updates for your team and see Cozmo react! Cozmo should roll off the charger, raise
+            and lower his lift, show an image on his face and speak the in-game update.
+'''
+
+import json
+import queue
+import sys
+sys.path.append('../')
```

Comment:
Minor/For-later: i assume that's specifically for importing flask_helpers, might be better next to it. We should look at reworking examples folders in general to make it easier to understand and the order to try them in etc.